package.json

```diff
@@ -35,6 +35,7 @@
     "react": "0.12.2",
     "serve-static": "1.9.2",
     "shelljs": "0.4.0",
+    "sinon": ">=1.4.0 <2",
```

Comment:
@jh-bate @jebeck I'm fine with choosing a specific version. We can choose the latest one it installs right now (1.15.3). The reason I added it this way was because this specific dependency was being pulled in implicitly (rather than explicitly) from the inclusion of sinon-chai (and implicit peer dependencies will be deprecated in the future).

Is this something you want me to do?